src/csv2docx/csv2docx.py

```diff
@@ -30,17 +30,19 @@ def create_unique_name(filename: str, path: Path) -> Path:
     if filepath.exists():
         # Count available files with same name
         counter = len(list(path.glob(f"{filepath.stem}*docx"))) + 1
-        filepath = f"{path}/{filepath.stem}_{counter}.docx"
+        filepath = path / f"{filepath.stem}_{counter}.docx"
```

Comment:
One example where static type analysis succeeds. Here we assign filepath to a string, but we expect a `path` to be returned from the method 👍